src/extension.ts

```diff
@@ -268,14 +268,15 @@ export function deactivate(): void {
 }
 
 // tslint:disable-next-line:no-any
-function initCommand<T>(extensionContext: vscode.ExtensionContext, commandId: string, callback: (context?: T, ...rest: any[]) => void): void {
+function initCommand(extensionContext: vscode.ExtensionContext, commandId: string, callback: (...args: any[]) => void): void {
     // tslint:disable-next-line:no-any
-    initAsyncCommand(extensionContext, commandId, async (context?: T, ...remaining: any[]) => callback.apply(null, [context].concat(remaining)));
+    initAsyncCommand(extensionContext, commandId, async (...args: any[]) => Promise.resolve(callback(...args)));
```

Comment:
Since this is an async arrow function - does it need to be `await Promise.resolve(`?

I don't think this would functionally affect the user, but we track how long commands take in our telemetry

Reply:
That's a valid point. Will remove.